src/routes/safe/components/Apps/components/ConfirmTxModal/ReviewConfirm.tsx

```diff
@@ -101,7 +101,7 @@ export const ReviewConfirm = ({
     () => (isMultiSend ? '0' : txs[0]?.value && parseTxValue(txs[0]?.value)),
     [txs, isMultiSend],
   )
-  const operation = useMemo(() => (isMultiSend ? DELEGATE_CALL : CALL), [isMultiSend])
```

Comment:
This should remain a DELEGATE_CALL

Reply:
Thanks for the correction!

Reply:
https://github.com/gnosis/safe-contracts/blob/9b305a0f80da7f1107d1181f52c844f089557d05/contracts/libraries/MultiSend.sol#L27

The reason for this is that the multiseed call itself has to be a delegate call, but we don't want inner transactions to be a delegate call

Reply:
Makes sense 👍